tensorflow/python/keras/engine/training_v2.py

```diff
@@ -250,13 +250,15 @@ def fit(
       # Raise an error if steps_per_epoch isn't specified but the dataset
       # is infinite.
       # TODO(scottzhu): This check should probably happen in the adapter
-      training_utils.infer_steps_for_dataset(
+      inferred_steps = training_utils.infer_steps_for_dataset(
           model,
           training_dataset,
           steps_per_epoch,
           steps_name='steps_per_epoch',
           epochs=0)
-
+      
+      steps_per_epoch = inferred_steps if steps_per_epoch is None else steps_per_epoch
```

Comment:
`FAIL: Found 1 non-whitelisted pylint errors:
tensorflow/python/keras/engine/training_v2.py:260: [C0301(line-too-long), ] Line too long (86/80)`
can you please check this lint error

Reply:
Hi! Sorry for the late reply. Added some line-breaks according to Google Style Guide, hope it's good now.